CONTRIBUTING.md

```diff
@@ -89,6 +89,19 @@ Eclipse settings.
 It is forbidden to use _wildcard imports_ (e.g., `import static org.junit.jupiter.api.Assertions.*;`)
 in Java code.
 
+### 🚀 Code Transformation & Automated Rewriting
+*(Advanced Refactoring, Modernization, and Bulk Code Changes)*
+
+#### 🔧 Automated rewriting of source code to
+- **Refactor** safely (e.g., rename methods, migrate APIs)
+- **Modernize** (e.g., Java 8 → Java 17 features)
+- **Fix anti-patterns** (e.g., replace `Vector` with `ArrayList`)
+- **Enforce conventions** (e.g., JUnit's naming rules)
+
+The build system incorporates [Moderne](https://moderne.io/) rewrite capabilities for automated code transformations. These modifications are environment-driven enlisted in the [junitbuild.java-library-conventions.gradle.kts](gradle/plugins/common/src/main/kotlin/junitbuild.java-library-conventions.gradle.kts)
+
+You can use `gradle rewriteRun` to apply these changes.
+
```

Comment:
Again: please restore the original wording and structure.

Keep "The build system..." block and the "You can use `gradle rewriteRun` to apply these changes." in the `Code` section. No extra structure, no emojis, no "ads/examples for Rewrite". Just blend in into the existing style of the document.

Additionally, don't force-push. I'd like to see incremental changes between commits.